tests/unit/tls_test.cc

```diff
@@ -182,11 +185,14 @@ SEASTAR_TEST_CASE(test_x509_client_with_system_trust_and_priority_strings) {
         "SECURE128:-VERS-TLS1.0:+COMP-DEFLATE",
         "SECURE128:+SECURE192:-VERS-TLS-ALL:+VERS-TLS1.2"
     });
+    auto set_priority_string = [](tls::credentials_builder & b, const sstring & prio) {
+        b.set_priority_string(prio);
+    };
```

Comment:
There's now 8 of these wrappers, perhaps replace it with something at file scope? Not sure how that works out with tls 1.3, though.
```c++
static void set_priority_string(tls::credentials_builder & b, const sstring & prio) {
#ifndef SEASTAR_WITH_TLS_OSSL
    b.set_priority_string(prio);
#else
    b.set_cipher_string(prio);
#endif
};
```

Reply:
> Not sure how that works out with tls 1.3, though.

Will probably craft a specific test for that